pkgs/by-name/cy/cyphernetes/package.nix

```diff
@@ -0,0 +1,82 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nodejs,
+  nix-update-script,
+  pnpm,
```

Comment:
Most likely you need to change `pnpm` to `pnpm_9`. Since #371832, it points to `pnpm_10` instead of `pnpm_9`.